deploy/pivot_role/pivotRoleCDK/dataall_base_infra.py

```diff
@@ -0,0 +1,827 @@
+                                         resources=["*"],
+                                         conditions={"StringEqualsIfExists": {"aws:ResourceTag/dataall": "true"}}
+                                     ),
+                                     # Dev Tools
+                                     iam.PolicyStatement(
+                                         sid="DevTools0", effect=iam.Effect.ALLOW,
+                                         actions=["cloudformation:ValidateTemplate"],
+                                         resources=["*"]
+                                     ),
+                                     iam.PolicyStatement(
+                                         sid="DevTools1", effect=iam.Effect.ALLOW,
+                                         actions=[
+                                             "secretsmanager:CreateSecret",
+                                             "secretsmanager:DeleteSecret",
+                                             "secretsmanager:TagResource",
+                                             "codebuild:DeleteProject"
+                                         ],
+                                         resources=["*"],
+                                         conditions={"StringEquals": {"aws:ResourceTag/dataall": "true"}}
+                                     ),
+                                     iam.PolicyStatement(
+                                         sid="DevTools2", effect=iam.Effect.ALLOW,
+                                         actions=[
+                                             "codebuild:CreateProject",
+                                             "ecr:CreateRepository",
+                                             "ssm:PutParameter",
+                                             "ssm:AddTagsToResource"
+                                         ],
+                                         resources=["*"],
+                                         conditions={"StringEquals": {"aws:RequestTag/dataall": "true"}}
+                                     ),
+                                     iam.PolicyStatement(
+                                         sid="CloudFormation", effect=iam.Effect.ALLOW,
+                                         actions=[
+                                             "cloudformation:DescribeStacks",
+                                             "cloudformation:DescribeStackResources",
+                                             "cloudformation:DescribeStackEvents",
+                                             "cloudformation:DeleteStack",
+                                             "cloudformation:CreateStack",
+                                             "cloudformation:GetTemplate",
+                                             "cloudformation:ListStackResources",
+                                             "cloudformation:DescribeStackResource"
+                                         ],
```

Comment:
Need for resource  `!Sub 'arn:aws:cloudformation:*:${AWS::AccountId}:stack/*/*'` to match pivotRole yaml and for pipeline github templates to work

Reply:
I added it but maybe we need to restrict the deployment in github templates and allow only stacks prefixed

Reply:
yes yes we can call out in documentation instead of being overly-permissive in the pivot role